lib/ui/search.dart

```diff
@@ -26,10 +30,25 @@ class SearchWidgetState extends State<SearchWidget> {
   final DateFormat _dateFormat = DateFormat("M/d/yyyy");
   DateTime _start = DateTime.now();
   DateTime _end = DateTime.now();
+  String _keyword = "";
+  TextEditingController keywordInput = TextEditingController();
+
+  // Apply and passed in search parameters to the filters
+  void applyFilters() {
+    if (this.widget.parameters.isEmpty) return;
+    SearchCriteria filters = SearchCriteria.withList(this.widget.parameters);
```

Comment:
Canonical Dart prefers using `final` over explicit typing.
See: https://dart.dev/guides/language/effective-dart/usage#do-follow-a-consistent-rule-for-var-and-final-on-local-variables